src/page-multi-facility/FacilityPage.tsx

```diff
@@ -1,14 +1,16 @@
-import React from "react";
+import React, { useContext, useEffect, useState } from "react";
```

Comment:
Some of these aren't used so I'll delete.  Not sure why the linter didn't pick these up